packages/models-library/src/models_library/api_schemas_catalog/services.py

```diff
@@ -310,3 +310,28 @@ class Config:
 ]
 
 ServiceResourcesGet: TypeAlias = ServiceResourcesDict
+
+
+class ServiceUpdateV2(BaseModel):
+    name: str | None = None
+    thumbnail: HttpUrl | None = None
+
+    description: str | None = None
+    version_display: str | None = None
+
+    deprecated: datetime | None = None
+
+    classifiers: list[str] | None = None
+    quality: dict[str, Any] = {}
+
+    access_rights: dict[GroupID, ServiceGroupAccessRightsV2] | None = None
+
+    class Config:
+        extra = Extra.forbid
```

Comment:
ah so in the end the forbid comes back ? ;)
it can help with debugging sometimes...

Reply:
Sometimes is useful. IMO this is one of the cases.